recipes/yum_requirements.txt

```diff
@@ -0,0 +1,6 @@
+mesa-libGL-devel
```

Comment:
Looks like this file not being in the recipe dir broke staged recipes. We should add something to smithy to lint for this. cc @ocefpaf